src/Mado/QueryBundle/Queries/Objects/Operator.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace Mado\QueryBundle\Queries\Objects;
```

Comment:
Can we merge this object with Dictionary/Operators or create new one?

Reply:
I'll remove Dictionary\Operators and I'll keep only `Mado\QueryBundle\Dictionary`.

Reply:
👍